### PR TITLE
Spelling correction in respect to function / variable names

### PR DESCRIPTION
**Pull request by albert-github**
As indicated in #7041 (Spelling corrections) there were still some open corrections in respect to variables / functions, these have been addressed here.

### CODE REVIEW

File: Polygon_mesh_processing/include/CGAL/Polyhedral_envelope.h [hunk 1]
```diff
@@ -1046,6 +1046,9 @@ struct Polyhedral_envelope {
     if (j == 2 && i == facesize - 1) return true;
     return false;
   }
+  CGAL_DEPRECATED bool
+  is_two_facets_neighbouring(const unsigned int & pid, const unsigned int &i, const unsigned int &j)const
+  { is_two_facets_neighboring(pid, i, j); }
```
Comment (sloriot) [inline Polygon_mesh_processing/include/CGAL/Polyhedral_envelope.h:1051]:
```suggestion
  { return is_two_facets_neighboring(pid, i, j); }
```

File: Arrangement_on_surface_2/include/CGAL/Curved_kernel_via_analysis_2/gfx/Curve_renderer_2.h [hunk 2]
```diff
@@ -2258,6 +2258,11 @@ bool compute_double_approx(int var, const NT& l_, const NT& r_,
     pix.yv = CGAL::to_double(engine.y_min + y*engine.pixel_h);
     return ret;
 }
+/*! \copydoc test_neighborhood
+ *  \deprecated please use #test_neighborhood */
+CGAL_DEPRECATED bool test_neighbourhood(Pixel_2& pix, int dir, int& new_dir)
+{ return test_neighborhood(pix, new_dir)' }
```
Comment (sloriot) [inline Arrangement_on_surface_2/include/CGAL/Curved_kernel_via_analysis_2/gfx/Curve_renderer_2.h:2264]:
```suggestion
{ return test_neighborhood(pix, new_dir); }
```

Comment (albert-github) [reply Arrangement_on_surface_2/include/CGAL/Curved_kernel_via_analysis_2/gfx/Curve_renderer_2.h]: Oops (for both), thanks for testing and fixing. 
(The problem is that I don't have all packages installed to do a compilation.)

**Comment by afabri** [comment]
The `CGAL_DEPRECATED`  [here](https://github.com/CGAL/cgal/pull/7186/files#diff-e2105006ee2b8d197607db0abf21886066d752f1ecc54d141c3435658f67fef1R36) makes todays testsuite red.

File: Jet_fitting_3/examples/Jet_fitting_3/PolyhedralSurf_rings.h [hunk 3]
```diff
@@ -78,6 +82,14 @@ push_neighbours_of(Vertex * start, int ith,
   }
 }
 
+CGAL_DEPRECATED template < class TPoly , class VertexPropertyMap>
+void T_PolyhedralSurf_rings <TPoly, VertexPropertyMap>::
```
Comment (afabri) [inline Jet_fitting_3/examples/Jet_fitting_3/PolyhedralSurf_rings.h:86]:
```suggestion
template < class TPoly , class VertexPropertyMap>
CGAL_DEPRECATED void T_PolyhedralSurf_rings <TPoly, VertexPropertyMap>::
```

File: Ridges_3/examples/Ridges_3/PolyhedralSurf_rings.h [hunk 4]
```diff
@@ -89,6 +92,13 @@ push_neighbours_of(const Vertex_const_handle start, const int ith,
   }
 }
 
+CGAL_DEPRECATED template < class TPoly >
+void T_PolyhedralSurf_rings <TPoly>::
+push_neighbours_of(const Vertex_const_handle start, const int ith,
```
Comment (afabri) [inline Ridges_3/examples/Ridges_3/PolyhedralSurf_rings.h:97]:
```suggestion
template < class TPoly >
CGAL_DEPRECATED void T_PolyhedralSurf_rings <TPoly>::
push_neighbours_of(const Vertex_const_handle start, const int ith,
```

File: Ridges_3/examples/Ridges_3/PolyhedralSurf_rings.h [hunk 5]
```diff
@@ -89,6 +92,13 @@ push_neighbours_of(const Vertex_const_handle start, const int ith,
   }
 }
 
+template < class TPoly >
+CGAL_DEPRECATED void T_PolyhedralSurf_rings <TPoly>::
```
Comment (afabri) [inline Ridges_3/examples/Ridges_3/PolyhedralSurf_rings.h:96]:
```suggestion
template < class TPoly >
void T_PolyhedralSurf_rings <TPoly>::
```

File: Ridges_3/test/Ridges_3/PolyhedralSurf_rings.h [hunk 6]
```diff
@@ -30,7 +30,10 @@ class T_PolyhedralSurf_rings
 
   //i >= 1; from a start vertex on the current i-1 ring, push non-visited neighbors
   //of start in the nextRing and set indices to i. Also add these vertices in all.
-  void push_neighbours_of(const Vertex_const_handle start, const int ith,
+  void push_neighbors_of(const Vertex_const_handle start, const int ith,
+                         std::vector < Vertex_const_handle > &nextRing,
+                         std::vector < Vertex_const_handle > &all);
+  CGAL_DEPRECATED void push_neighbours_of(const Vertex_const_handle start, const int ith,
```
Comment (afabri) [inline Ridges_3/test/Ridges_3/PolyhedralSurf_rings.h:36]:
```suggestion
                         std::vector < Vertex_const_handle > &all);
  void push_neighbours_of(const Vertex_const_handle start, const int ith,
```

File: Ridges_3/examples/Ridges_3/PolyhedralSurf_rings.h [hunk 7]
```diff
@@ -30,7 +30,10 @@ class T_PolyhedralSurf_rings
 
   //i >= 1; from a start vertex on the current i-1 ring, push non-visited neighbors
   //of start in the nextRing and set indices to i. Also add these vertices in all.
-  void push_neighbours_of(const Vertex_const_handle start, const int ith,
+  void push_neighbors_of(const Vertex_const_handle start, const int ith,
+                         std::vector < Vertex_const_handle > &nextRing,
+                         std::vector < Vertex_const_handle > &all);
+  CGAL_DEPRECATED void push_neighbours_of(const Vertex_const_handle start, const int ith,
                           std::vector < Vertex_const_handle > &nextRing,
```
Comment (afabri) [inline Ridges_3/examples/Ridges_3/PolyhedralSurf_rings.h:37]:
```suggestion
  void push_neighbours_of(const Vertex_const_handle start, const int ith,
                          std::vector < Vertex_const_handle > &nextRing,
```

File: Jet_fitting_3/examples/Jet_fitting_3/PolyhedralSurf_rings.h [hunk 8]
```diff
@@ -25,7 +25,11 @@ template < class TPoly , class VertexPropertyMap> class T_PolyhedralSurf_rings
...
   //i >= 1; from a start vertex on the current i-1 ring, push non-visited neighbors
   //of start in the nextRing and set indices to i. Also add these vertices in all.
-  static void push_neighbours_of(Vertex * start, int ith,
+  static void push_neighbors_of(Vertex * start, int ith,
+                          std::vector < Vertex * >&nextRing,
+                          std::vector < Vertex * >&all,
+                          VertexPropertyMap& vpm);
+  CGAL_DEPRECATED static void push_neighbours_of(Vertex * start, int ith,
                           std::vector < Vertex * >&nextRing,
                           std::vector < Vertex * >&all,
```
Comment (afabri) [inline Jet_fitting_3/examples/Jet_fitting_3/PolyhedralSurf_rings.h:34]:
```suggestion
  static void push_neighbours_of(Vertex * start, int ith,
                          std::vector < Vertex * >&nextRing,
                          std::vector < Vertex * >&all,
```

**Comment by albert-github** [comment]
@afabri thanks for looking into it and correcting it.

**Comment by sloriot** [comment]
Successfully tested in CGAL-5.6-Ic-160

**Pull request merged**